femmt/component.py

```diff
@@ -428,7 +429,7 @@ def set_winding_windows(self, winding_windows: list[WindingWindow]):
                         vww.turns.append(0)
 
         # Default values for global_accuracy and padding
-        self.mesh_data = MeshData(0.5, 1.5, mu_0, self.core.core_inner_diameter, self.core.window_w, self.windings)
+        self.mesh_data = MeshData(0.9, 1.5, mu_0, self.core.core_inner_diameter, self.core.window_w, self.windings)
```

Comment:
What is the reason for this mesh change?

Reply:
to reduce the simulation's duration. It takes long time with more mesh accuracy